config/index.js

```diff
@@ -70,7 +70,10 @@ module.exports = {
   SERVER_HOST,
   PORT: process.env.PORT || 3000,
   LOG_LEVEL: process.env.LOG_LEVEL || (IS_DEV ? 'debug' : 'error'),
-  NO_CACHE: process.env.CACHE_ASSETS ? false : IS_DEV,
```

Comment:
I quick poke in the history shows it was [used for nunjucks config](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/commit/ed9647e7c4a5b9167fe240e72ae71b2f85221b39#diff-54e92bb4c2374556f8a312c85f573102418d9ccc3daa5a2e962f53f58a6862f0L16), which has since [been updated](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/commit/26864a3dbba01d700e93723850a0575bc48d1ebf) to work based on whether the app is in development or not. So this is safe to rename :)